### DISCUSSION TITLE
A few questions about specifying a Video Engine 关于指定 Video Engine 的几个问题

### DISCUSSION THREAD

**Discussion by DOSforever** [SubtitleEdit/subtitleedit#8381] [Q&A]
由于 Sutitle Edit 无法自行下载 mpv lib ，所以我根据 URI 单独另行下载了 libmpv2-64.zip ，那么下载后应该把 libmpv2-64.zip 或解压后的 libmpv-2.dll 文件后应该放在 Subtitle Edit 的哪个目录中才能生效？
如果指定 VLC media player ，可以提供 portable 版的路径，但为什么对 MPC-HC 却没有提供 portable 版的路径选择呢？

Since Sutitle Edit can't download the mpv lib by itself, so I downloaded libmpv2-64.zip separately according to the URI, then which directory of Subtitle Edit should I put the libmpv2-64.zip or the unzipped libmpv-2.dll file in to make it effective?
If I specify VLC media player, it can provide the path of portable version, but why there is no path of portable version for MPC-HC?

**Comment by niksedk** [comment]
libmpv files needs to go into the `SE data folder` - press `Ctrl+alt+shift+d` in the main SE window to open this folder

**Comment by DOSforever** [comment]
我用的是 SE405.zip portable version ，当我按 Ctrl+alt+shift+d 时打开的就是 SubtitleEdit.exe 文件所在的 SE405 主目录，我把 libmpv-2.dll 放在该目录中，启动 Subtitle Edit ，在 Video Player 选项中点选了 mpv ，此时 Download mpv lib 按钮仍可用，点 OK 退出 Settings ，打开一包含字幕流的视频文件，点播放按钮无任何反应。但是如果选择的是 VLC 则可以播放。

I am using the SE405.zip portable version. When I press Ctrl+alt+shift+d, it opens the main directory of the SubtitleEdit.exe file. I placed the libmpv-2.dll in that directory. I started Subtitle Edit, selected mpv in the Video Player option. At this point, the "Download mpv lib" button is still available. I clicked OK to exit Settings, opened a video file containing subtitle streams, and clicked the play button with no response. However, if VLC is selected, it can play.

**Comment by niksedk** [comment]
It should be the correct folder.

What Os are you using and is it 64-bit?
You cannot select "mpv" via the radio button?
![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/e7b75a51-7890-4202-b1a2-e0e841764494)

**Comment by DOSforever** [comment]
Yes, I'm using Windows 7 64-bit Home Edition, The SE405 is also a 64-bit version.
I can select "mpv" via the radio button same as your screenshots.
